Training1/OperatorOverloading.DBL/CurrencyConverter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Interface;
+using System.IO;
+namespace OperatorOverloading.DBL
+{
+    public class CurrencyConverter : ICurencyConverter
+    {
+        private string _jsonFile;
+        public string JsonFile
+        {
+            get
+            {
+                return _jsonFile;
+            }
+            set
```

Comment:
Shouldn't this be `private`? Steps to break your code
* Construct object 
* Set `JsonFile` to `null`